frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -522,6 +522,23 @@ export function Experiment(): JSX.Element {
                                     </Col>
                                 </Row>
                             )}
+                        {showWarning &&
+                            experimentResults &&
+                            ((experimentInsightType === InsightType.TRENDS && !areCountResultsSignificant) ||
+                                (experimentInsightType === InsightType.FUNNELS &&
+                                    !areConversionResultsSignificant)) && (
+                                <Row align="middle" className="not-significant-results">
+                                    <Col span={19} style={{ color: '#f96132' }}>
+                                        Experiment results are <b>not significant</b>. You shouldn't end your experiment
```

Comment:
```suggestion
                                        Experiment results are <b>not yet significant</b>. We don't recommend ending your experiment until results are significant.
```

Reply:
`We recommend not ending` vs `We don't recommend ending` ?

Reply:
Hmm unsure, I think you're right, might be worth asking @joethreepwood for some quick input

Reply:
I'd change this line to the following: 

```suggestion
                                        Your results are <b>not conclusive</b>. We don't recommend ending this experiment yet. 
```
or 

```suggestion
                                        Your results are <b>not statistically significant</b>. We don't recommend ending this experiment yet. 
```

'Experiment results are not significant' feels a bit tricky, as 'significant' can have multiple meanings and the tense reads a little off. It could be that the results are statistically significant, but not materially significant for example. Or vice versa. 

We'd also change the positive line, obviously. 

```suggestion
                                        Your results are <b>statistically significant</b>. You can end this experiment now or let it run to completion.
```

Reply:
Makes sense, done!